assets/data/peopleParts/drauvenArmour01.json

```diff
@@ -0,0 +1,20 @@
+{
+	"type": "extra",
+	"extraSlot": "zzz_drauven06_Armour",
+	"requireAspects": ["themePiece_drauven_torso"],
+
+	"groups": [
+		{"groupIndex": 0, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 1, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 2, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1},
+		{"groupIndex": 3, "male": true, "maleWeight": 1, "female": true, "femaleWeight": 1}
+	],
+  "layers": [
+		{
+			"name": "outfitVariation1",
```

Comment:
🤔 It might be good to break the outfits down into the different pieces like we had before - the arm bands for the hunter being their own layers, etc. To make that work we should probably have each armour piece set up to only apply to the appropriate rig using the `requireAspects` field.

Also, since the preview only shows head layers and also shows all head layers regardless of what the `if(No)OwnerAspects` is set to, it would probably be good to set a head layer with an `ifOwnerAspect: _NEVER_` or something like that so players can see what they're choosing.